test/scraper/schleswigholstein_landtag_scraper_overview_test.rb

```diff
@@ -0,0 +1,72 @@
+require 'test_helper'
+
+class SchleswigholsteinLandtagScraperOverviewTest < ActiveSupport::TestCase
+  def setup
+    @scraper = SchleswigHolsteinLandtagScraper
+    @baseurl = "http://lissh.lvn.parlanet.de/cgi-bin/starfinder/0?path=lisshfl.txt&id=fastlink&pass=&search=((dtyp%3dkleine+and+WP%3d18))"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.